### ISSUE TITLE
Triage and fix TODOs, FIXMEs and XXXs

### ISSUE THREAD

**Issue by lzaoral** [openscanhub/openscanhub#103]
This project now contains the following FIXMEs, TODOs and XXXs obtained using `git grep -IPi '\b(TODO|XXX|FIXME)\b'`.  This issue serves as a tracking bug for all of them (fixed instances are marked using ✅): 

- [x] ``.github/workflows/lint.yml:23:          # TODO: This should be moved to `make lint` once we  migrate to GitHub``
  * https://github.com/openscanhub/openscanhub/pull/188
- [x] `README.md:51:TODO`
  * https://github.com/openscanhub/openscanhub/pull/196
- [ ] `README.md:55:TODO`
- [ ] `osh.spec:128:# TODO: Remove this when we migrate to newer Python packaging macros which`
- [ ] `osh/client/osh-cli:40:        # TODO: Uncomment in the future.  For now, we should give users some`
- [ ] `osh/common/csmock_parser.py:266:            # XXX: this destroys all containers we have access to, related or not`
- [ ] `osh/hub/errata/scanner.py:219:            # TODO: make this configurable`
- [ ] `osh/hub/errata/scanner.py:223:            # TODO: make this configurable`
- [ ] `osh/hub/errata/scanner.py:468:        # FIXME: ideally rewrite the code to stuff all input-related info to "source" (e.g. builds,`
- [ ] `osh/hub/osh_xmlrpc/worker.py:135:        # FIXME: hard-coded at two places for now`
- [ ] `osh/hub/osh_xmlrpc/worker.py:160:            # FIXME: hard-coded at two places for now`
- [ ] `osh/hub/scan/models.py:899:        # FIXME: this will not detect equally sized sets of non-matching items`
- [ ] `osh/hub/scan/models.py:906:                    # FIXME: move this list to the database to ease updates`
- [ ] `osh/hub/scan/models.py:1233:# TODO: We should replace this key with a One-to-Many releation with Analyzer.`
- [ ] `osh/hub/scan/notify.py:57:                # XXX: hardcoded`
- [ ] `osh/hub/scan/notify.py:252:                date.strftime('%Y-%m-%d %H:%M:%S'),  # TODO: add %Z - TZ`
- [x] `osh/hub/scan/task_urls.py:13:         # FIXME: https://gitlab.cee.redhat.com/covscan/covscan/-/issues/125`
  * https://github.com/openscanhub/openscanhub/pull/240
- [ ] `osh/hub/scan/views.py:59:    # XXX: only for compatibility with old API`
- [ ] `osh/hub/scan/xmlrpc_helper.py:45:    # TODO: create separate function`
- [ ] `osh/hub/settings.py:47:# TODO: By default, Django 3 uses BigAutoField but we must remain compatible`
- [ ] `osh/hub/stats/stattypes.py:496:    # TODO: Django 4 introduced default kwarg for the value of an empty Sum,`
- [ ] `osh/hub/waiving/forms.py:38:        # FIXME This is actually happening, *somehow*; this doesn't work:`
- [ ] `osh/hub/waiving/views.py:235:    # FIXME: 5 columns doesn't work`
- [ ] `osh/hub/waiving/views.py:493:    context = context.copy()  # TODO: check if .copy() is really needed`
- [x] `osh/worker/tasks/task_errata_diff_build.py:64:                # FIXME: workaround it here for now`
  * https://github.com/openscanhub/openscanhub/pull/72
- [x] `scripts/cli_sanity_test.sh:11:    # TODO: use an array to store test tasks and abort them all here`
  * https://github.com/openscanhub/openscanhub/pull/188
- [x] `scripts/cli_sanity_test.sh:88:                # TODO: get this path working`
  * https://github.com/openscanhub/openscanhub/pull/188
- [x] `scripts/cli_sanity_test.sh:137:    # TODO: test cancel-tasks and find-tasks`
  * https://github.com/openscanhub/openscanhub/pull/188

**Comment by kdudka** [comment]
I do not think we need to fix these issues proactively (and the above list  might already be outdated).
